warehouse/malware/checks/base.py

```diff
@@ -10,18 +10,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from warehouse.malware.models import MalwareCheck, MalwareCheckState
+from warehouse.malware.models import MalwareCheck, MalwareCheckState, MalwareVerdict
 
 
 class MalwareCheckBase:
     def __init__(self, db):
         self.db = db
         self._name = self.__class__.__name__
         self._load_check()
```

Comment:
Nit: Maybe move this attr above the `_load_check()` call, just for consistency with the other attrs?

Reply:
I _could_ do this, but I use `self._name` in `_load_check` to look up the id.

Reply:
Ah! I missed that, sorry. In that case this LGTM 🙂